onadata/libs/utils/common_tools.py

```diff
@@ -104,3 +104,28 @@ def get_response_content(response):
         contents = response.content
 
     return contents
+
+
+def json_stream(data, json_string):
```

Comment:
for the second argument let's have the name indicate it's a function, e.g. `item_transform_fn`